docs/usage/Cluster.md

```diff
@@ -17,6 +17,27 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
     * **Cluster name**: May contain lower-case letters (a-z) and numbers (0-9). Must not contain spaces or international characters.
```

Comment:
These might looks better formatted as a table

Reply:
Probably, but I'm trying to keep it the same as the downstream docs which have a bulleted list

Reply:
ACK, that works.